content/docs/reference/types/object.mdx

```diff
@@ -45,15 +45,15 @@ Ultimately there are 3 ways to configure this type...
 
 * With `list` set to `false`, the `fields` array can be used to wrap some data – this appears in the editor as a nested page. This can be used to organise the CMS, such as grouping options together, as in the gif below.
 
-![](https://res.cloudinary.com/forestry-demo/image/upload/v1727331124/tina-io/docs/Sep-26-2024_16-05-37_d2ol8w.gif)
+<WebmEmbed embedSrc="public/img/docs/reference/rodmap-grid.webm"/>
```

Comment:
**issue**: Image not shown due to 'public' prefix.

Reply:
Done resolved